build.ps1

```diff
@@ -4,15 +4,20 @@ This is a Powershell script to bootstrap a Cake build.
 .DESCRIPTION
```

Comment:
I'm going to ignore changes to this file. 

Warning: the next time we get the latest version from the Cake repo all our changes will be lost.

Reply:
Acknowledged!